routers/repo/http.go

```diff
@@ -298,6 +298,19 @@ func HTTP(ctx *context.Context) {
 		}
 	}
 
+	if isWiki {
+		// Ensure the wiki is enabled before we allow access to it
+		if _, err := repo.GetUnit(models.UnitTypeWiki); err != nil {
```

Comment:
The `else` is missing for this condition.

Reply:
I don't understand

Reply:
Oh, I see, `repo.GetUnit` will return an error if no unit is configured. Don't mind me then.